src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css

```diff
@@ -14,97 +14,10 @@ a, .btn-link {
     border-color: #1861ac;
 }
 
-app {
-    position: relative;
-    display: flex;
-    flex-direction: column;
-}
```

Comment:
At first I thought it was a mistake to remove `app` in the wasm case because we're still using it. But then I realised you didn't remove it from `index.html`, so this is OK.

We *could* change `index.html` to have `<div id="app">` instead of `<app>` and update the selector in `Program.cs`, but I'm not massively bothered either way, so up to you whether we do that now or some other time in the future.

Reply:
I did remove app in favor of using a div